Solutions/MCBSTM32F400/TinyCLR/scatterfile_tinyclr_mdk.xml

```diff
@@ -50,7 +50,7 @@
 
     <!-- DAT region goes into external NOR Flash -->
     <Set Name="Data_BaseAddress"        Value="%Config_BaseAddress% + %Config_Size%"/>
-    <Set Name="Data_Size"               Value="0x00020000"/><!-- 128KB -->
+    <Set Name="Data_Size"               Value="0x00030000"/><!-- 128KB -->

```

Comment:
0x00030000 != 128KB, please be sure comments are correctly updated to reflect changes!